src/i18n/strings/ar.json

```diff
@@ -0,0 +1,16 @@
+{"أضف عنوان بريد إلكتروني أعلاه لتهيئة إشعارات البريد الإلكتروني": "أجوتر ون أدريس إمايل بور لا كونفيغوراتيون ديس نوتيفيكاتيونس بار إمايل"،
```

Comment:
You are using chars other than `,` as the entry seperator, this will not work :/